DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_code_split/src/simple.cpp

```diff
@@ -0,0 +1,84 @@
+#include <sycl/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <iostream>
+
+using namespace sycl;
+constexpr access::mode sycl_read = access::mode::read;
+constexpr access::mode sycl_write = access::mode::write;
+
+class SimpleAdd;
+class SimpleMul;
+
+void kernelAdd(const float a, const float b, float &c) {
+
+  {
+#if FPGA_SIMULATOR
+    auto selector = sycl::ext::intel::fpga_simulator_selector_v;
+#elif FPGA_HARDWARE
+    auto selector = sycl::ext::intel::fpga_selector_v;
+#else // #if FPGA_EMULATOR
+    auto selector = sycl::ext::intel::fpga_emulator_selector_v;
+#endif
+
+    queue q(selector);
+    buffer<float, 1> bufferA(&a, 1);
```

Comment:
Variables should be snake_case
```suggestion
    buffer<float, 1> buffer_a(&a, 1);
```